mne/io/artemis123/artemis123.py

```diff
@@ -32,6 +35,13 @@ def read_raw_artemis123(input_fname, preload=False, verbose=None):
         on the hard drive (slower, requires less memory).
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
+    pos_fname : str or None (default None)
+        If not None, load digitized head points from this file
+    head_loc : bool (default True)
+        If True attempt to perform initial head localization (compute initial
+        evice to head coordinate transform) using HPI coils. If no
+        HPI coils are in info['dig'] hpi coils are assumed to be in canonical
+        order of fiducial points (nas, rpa, lpa).
```

Comment:
It should probably be `None` but let's save it for another PR, since it's in `_empty_info` I agree we'd need to change it there (and see what the problems would be).